addon/styles/select.less

```diff
@@ -359,8 +348,7 @@
 
   .dropdown-menu {
     // This causes stress
-    min-width: 200px;
-    max-width: 600px;
+    width: 200px;
```

Comment:
That seems weird.

@LydiaBarnes thoughts?

Reply:
@mixonic Hmm what is the need for this change? Selects all over the app would be affected and theoretically not be able to grow, so words might get ellipsed too early, etc. A lot of them have widths hacked over them in Iverson but its hard to tell the full effect.

Reply:
Thanks for noting this @cyril-sf. @LydiaBarnes the issue is that when scrolling if wider content is present than was present during initial render, the box becomes wider. this can be pretty surprising.

I haven't confirmed what the existing behavior is, but I don't believe we want the box to be changing widths as the user scrolls.

* If this is already the behavior and the "width hacks" in Iverson already handle it, then I can just remove this change.
* If the existing behavior is that the initial width of the drop-down is retained, then I'll need to do some extra work to make that happen.

What do you think?

Reply:
Ok I poked around at Iverson selects. Looks like we overwrite width to be 100%. I guess through some sorcery everything still works out. So theoretically if you change it here, it won't effect anything since it gets overwritten globally there.
I would make it 300 though just to be super safe, since we also globally add a min-width: 300px on them.
Eventually this nightmare select will go away right? 😀

Reply:
I'll confirm the behavior hasn't changed, or discover exactly how it has. I don't think a width of 100% would stomp the `max-width` here? I'll check.

Reply:
Ok, I did a visual side-by-side comparison to make sure things look the same.

I'm going to leave the width value. With list view if all items were < 200px wide when you scrolled down to them they would be truncated. In this implementation if there is no width specified here then the select will widen and narrow according to the content (within the min/max widths).

I think the better default here is to just pin to 200px wide. @LydiaBarnes is correct that often a width of 100% is set in Iverson, and override this, so the default is largely moot.